File: PHP/program/Mahasiswa.php
<?php

include "SivitasAkademik.php";

class Mahasiswa extends SivitasAkademik
{
    private $nim = "";
    private $prodi = "";
    private $fakultas = "";

    public function __construct($nik = "", $nama = "", $jenkel = "", $asalUniv = "", $emailEdu = "", $nim = "", $prodi = "", $fakultas = "")
    {
        parent::__construct($nik, $nama, $jenkel, $asalUniv, $emailEdu);
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    public function getNIM()
    {
        return $this->nim;
    }

    public function setNIM($nim)
    {
        $this->nim = $nim;
    }

    public function getProdi()
    {
        return $this->prodi;
    }

    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }

    public function getFakultas()
    {
        return $this->fakultas;
    }

    public function setFakultas()
    {
        $this->fakultas = $fakultas;
    }
}

?>

File: CPP/program/Mahasiswa.cpp
#include "SivitasAkademik.cpp"

class Mahasiswa : public SivitasAkademik
{
    private:

        string nim;
        string prodi;
        string fakultas;

    public:

        Mahasiswa(){}

        Mahasiswa(string nik, string nama, string jenkel, string asalUniv, string emailEdu, string nim, string prodi, string fakultas)
        {
            this->nik = nik;
            this->nama = nama;
            this->jenkel = jenkel;

            this->asalUniv = asalUniv;
            this->emailEdu = emailEdu;

            this->nim = nim;
            this->prodi = prodi;
            this->fakultas = fakultas;
        }

        ~Mahasiswa(){}

        string getNIM()
        {
            return nim;
        }

        void setNIM(string nim)
        {
            this->nim = nim;
        }

        string getProdi()
        {
            return prodi;
        }

        void setProdi(string prodi)
        {
            this->prodi = prodi;
        }

        string getFakultas()
        {
            return fakultas;
        }

        void setFakultas(string fakultas)
        {
            this->fakultas = fakultas;
        }
};